### DISCUSSION TITLE
Set up in new location with no computer?

### DISCUSSION THREAD

**Discussion by RamblerKevin** [rern/rAudio#1409] [Tips]
Hi,

Headed to a friends and was thinking about taking the pi machine running R Audio with me. How would i go about setting it up on their WIFI if i don't have access to a computer to edit the card inside with the WIFI info?

Thanks

**Comment by EarsofTin** [comment]
You can connect directly to the pi from a phone or tablet via the access point it generates. Then edit the wifi connection on the Networks page with your friends wifi info and reboot.

From the ReadMe:

Wi-Fi access point mode
If no wired network and no pre-configured Wi-Fi connections, Wi-Fi access point will be enabled on boot.
On client devices, select rAudio from Wi-Fi network list to connect.
On browser, open web user interface with URL raudio.local
Settings > Networks > Wi-Fi - search
Select access point to connect
Reboot
Browser refreshes when ready. (Manually refresh if it's too long.)

**Comment by saelden** [comment]
I, too, would be great if Wi-Fi access point mode worked.
